src/libraries/System.Runtime/ref/System.Runtime.cs

```diff
@@ -7357,7 +7357,7 @@ public partial class StreamWriter : System.IO.TextWriter
         public override System.Threading.Tasks.Task WriteAsync(char[] buffer, int index, int count) { throw null; }
         public override System.Threading.Tasks.Task WriteAsync(System.ReadOnlyMemory<char> buffer, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
         public override System.Threading.Tasks.Task WriteAsync(string? value) { throw null; }
-        public override void WriteLine(System.ReadOnlySpan<char> buffer) { }
+        public override void WriteLine(System.ReadOnlySpan<char> value) { }
```

Comment:
This introduces a difference from the base TextWriter.WriteLine, which takes "buffer".  Seems like either this should stay "buffer", or the base should change as well?

Reply:
We can change the implementation instead.  I'll do that.